app/views/scan_analytics/_index.html.haml

```diff
@@ -0,0 +1,69 @@
+:ruby
+  scan_data = @scan_data.present? ? @scan_data : scan_data
+  scan_data = JSON.parse(scan_data.data.to_json, object_class: OpenStruct)
+  @cwes = scan_data.cwe
+  version = scan_data.analysis_metrics.version.present? ? scan_data.analysis_metrics.version : nil
+  last_analysed = scan_data.analysis_metrics.build_date
+  loc = scan_data.analysis_metrics.loc
+  defect_density = scan_data.analysis_metrics.defect_density&.score
+  total_defects = scan_data.analysis_metrics.total_count
+  outstanding_count = scan_data.analysis_metrics.outstanding_count
+  fixed_count = scan_data.analysis_metrics.fixed_count
```

Comment:
Instead of assigning 7 variables here, assign a single variable `metrics = scan_data.analysis_metrics` and use `metrics.loc` etc. further down.